alibi/explainers/backends/pytorch/cfrl_base.py

```diff
@@ -3,16 +3,17 @@
 :py:class:`alibi.explainers.cfrl_base` for the Pytorch backend.
 """
 
-import torch
-import torch.nn as nn
-import torch.nn.functional as F
-from torch.utils.data import DataLoader, Dataset
 
 import os
 import random
-import numpy as np
 from typing import List, Dict, Callable, Union, Optional, TYPE_CHECKING
 
+import numpy as np
```

Comment:
Are these changes the result of `isort`?

Reply:
No, I did them myself, I think when I was implementing the `optional_import` pattern for the backends I made them but have since removed the backend `optional_import` stuff.